### ISSUE TITLE
I cant Uninstall Npcap and when i try to install it it instantly aborts 

### ISSUE THREAD

**Issue by OnceANut** [nmap/nmap#1762]
i just need to uninstall it and reinstall it to use wire shark on my network for school

**Comment by Jerry4013** [comment]
Same issue. I deleted the "program files/npcap" folder. Then I cannot install a new version.

**Comment by dmiller-nmap** [comment]
Thanks for reporting this. Are you using the latest version of Npcap? Npcap 0.994 and 0.995 have a known bug (#1626) that has been solved in later releases. To avoid a crash when uninstalling, first disable the `npcap` and `npf` services and reboot. Then uninstall or upgrade as usual.

**Comment by Jerry4013** [comment]
@dmiller-nmap Yes, I tried to update it to the newest version, but I already removed the folder before I knew I should have disable the service first. Now I cannot install the newest version because everytime it checks the regedit and I don't know how to remove the remainder record. Thanks!

**Comment by CDBridger** [comment]
I've installed Win10pcap to tide me over, but I would very much like to revert back to Npcap =/
